docs/gl_objects/issues.rst

```diff
@@ -225,3 +225,51 @@ Link issue ``i1`` to issue ``i2``::
 Delete a link::
 
     i1.links.delete(issue_link_id)
+
+Issues statistics
+=========================
+
+Reference
+---------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.IssuesStatistics`
+  + :class:`gitlab.v4.objects.IssuesStatisticsManager`
+  + :attr:`gitlab.issuesstatistics`
```

Comment:
Is it possible to do: `gitlab.issues_statistics` ?

Changing `issuesstatistics` to `issue_statistics`

Reply:
I've changed this and for now silently deprecated the existing `issuesstatistics` in Project. I guess we'd need a nice helper to make it easy to add deprecation warnings to any manager class or attribute.